src/coreclr/nativeaot/Runtime/windows/PalMinWin.cpp

```diff
@@ -1039,7 +1039,7 @@ UInt32_BOOL PalCloseHandle(HANDLE arg1)
     return ::CloseHandle(arg1);
 }
 
-void PalFlushProcessWriteBuffers()
+void minipal_memory_barrier_process_wide()
```

Comment:
Should this be deleted instead? I would expect the implementation to come from minipal.